docs/README.md

```diff
@@ -441,6 +441,90 @@ You can expand any of the messages that matches the filter to see the full stack
+
+Cloud operations suite provides **service oriented monitoring**, that means that we are configuring SLIs, SLOs and Burning Rates Alerts for a 'service'.  
+
+The first step in order to create SLO is to **ingest the data**, for GKE services  it comes out of the box, but you can also ingest additional data and [create custom metrics.](#Monitoring-and-logs-based-metrics)
+
+Then we need to **define our service**, Cloud Operations Sandbox' services are already detected since Istio automatically detects and creates services for us. But to demonstrate that you can create your own services, it also deploys custom services using [Terraform](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring).
+
+You can find all the services under [monitoring → services → Services Overview](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/svc-overview) , and you can create your own [custom service.](https://cloud.google.com/stackdriver/docs/solutions/slo-monitoring/ui/define-svc)
+
+![image](./images/user-guide/37-services-overview.png)  
+
+#### Services SLOs  
+
+The *predefined SLOs* are also deployed as part of [Terraform code](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/terraform/monitoring/04_slos.tf) and currently are for the mentioned custom services, the Istio service and Rating service.  
+
+**Custom services SLOs**
+``` 
+Custom service availability SLO: 90% of all non-4XX requests within the past 30 day windowed period return with 200 OK status 
+```
+
+``` 
+Custom service Latency SLO:  90% of requests that return 200 OK responses return in under 500 ms 
+```  
+To view the exiting SLOs, in the Services Overview screen choose the desired service.
+
+*For example for checkoutservice*:
+
+![image](./images/user-guide/47-choose-checkout-custom-service.png)
+
+![image](./images/user-guide/36-checkoutservice-overview.png)
+
+**Additional predefined SLOs:**
+
+```
+Istio service availability SLO: 90% of all non-4XX requests within the past 30 day windowed period return with 200 OK status
+```
+```
+Istio service latency SLO: 99% of requests that return 200 OK responses return in under 500 ms
+```
+```
+Rating service availability SLO: 99% of all non-4XX requests within the past 30 day windowed period  return with 200 OK status
+```
+```
+Rating service latency SLO: 99% of requests that return 200 OK responses return in under 175 ms
+```
+```
+Rating service's data freshness SLO: during a day 99.9% of minutes have at least 1 successful recollect API call
+```
+
+#### Configure your own SLIs and SLOs
+
+> **Remember**  Our scope to examine and measure our users' happiness is User journey, so in order to create the SLO you need to identify the most important ones to the business. Then we want to *identify the metrics* that are closest to the customer experience and ingest that data.  
```

Comment:
This could use some re-wording. Maybe:

> **Remember** The purpose of defining SLIs and SLOs is to improve your user's experience. Take some time to identify the most important [user journeys](https://en.wikipedia.org/wiki/User_journey) around your product, and use them to define which metrics to track.

Reply:
This change the meaning a bit, and this is one of the pitfalls with our customers. 

> **Remember** The purpose of defining SLIs and SLOs is to improve your user's experience, your SLOs scope is a [User journey](https://cloud.google.com/solutions/defining-SLOs#why_slos). Therefore your first step should be to *identify the most critical User Journey(CUJ)* to your business, then *identify the metrics* that measure your customer experience as closely as possible and *ingest* that data.  

WDYT?

Reply:
Yeah, I like that better, thanks!